app/Http/Controllers/RuleEngine/RuleController.php

```
@@ -0,0 +1,582 @@
+        return response()->json([
+            'message' => 'Rule activation status updated successfully',
+            'data' => $rule,
+        ]);
+    }
+
+    /**
+     * Get available fields, operators, and action types.
+     */
+    public function getOptions(): JsonResponse
+    {
+        $user = auth()->user();
```

PHPStan (2.1.15)

Call to an undefined method Illuminate\Contracts\Auth\Factory::user().


(method.notFound)